### DISCUSSION TITLE
Set runs for assessing the climate sensitivity in the Lscale branch

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#500] [Discussion about coupled runs]
### Purpose of the run

We want to start a set of runs to assess the climate sensitivity (4xco2 and 1%co2).
- The baseline for these runs will be 78 (Lscale code branch with faster dycore). 
- We want to use a spunup state for these runs. Unfortunately, 78 is only 49 years and it seems that it might be freezing over towards the end of the run. (@adamrher: can you do your spaghetti plot). 
- We are proposing to start these runs from the end of 75 (Lscale code branch with old dycore) as 75 doesn't seem to freeze over in Adam's plot. 
![Screen Shot 2024-02-15 at 4 32 12 PM](https://github.com/NCAR/amwg_dev/assets/9723220/9743ced9-64fd-43b3-9dce-731f071cd970)

- We will also add a 1xco2 run starting from the same state. 

In summary, we will start a total of 3 runs from the end of 75.
- 78b => BLT1850 starting from 75 at year 99
- 78b_4co2 => 4xco2 starting from 75 at year 99
- 78b_1%co2 => 1%co2  starting from 75 at year 99

In order to start these runs, I need Gustavo's configuration to include the restart from mom6.

### Description of the run

Enter Description

### Suffix in the casename

Suffix

### Namelist modifications

Namelist

### Source modifications

SourceMods

### Sandbox

Requested sandbox or tag

### Contact info

@cecilehannay @PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @brianpm, @swrneale, @islasimpson @dlawrenncar @gustavo-marques

### Any other relevant information

Enter relevant info

**Comment by cecilehannay** [comment]
@dlawrenncar: when I talked to you earlier, we decided to start from the end of 78 but I don't like the way the end of 78 looks.

**Comment by cecilehannay** [comment]
An alternative to this if we really want to start the 78_4xco2 from the 78 restart files (instead of the 75 restart files) is to do a pertlim on 78 and to rerun it hoping that the lab sea won't freeze. And then to start the the 4xco2 from the end of this pertlim run. But this will delay further the start of the 4xco2 runs. 

--- 

I will use the instructions from @gustavo-marques at: https://github.com/NCAR/amwg_dev/discussions/459#discussioncomment-8115445
to make sure the restart files from mom are read properly.

**Comment by adamrher** [comment]
@PeterHjortLauritzen did you add a ```if (things_are_going_well) freeze_the_LabSea``` statement to ur "science optimizations"?

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/4ea982b8-387c-40aa-8bed-f81d912eca9b)

hmm ...

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/73a52b73-ab34-4bf8-88bd-f45ff8a03987)

**Comment by PeterHjortLauritzen** [comment]
I agree with proceeding with @cecilehannay's original proposal (and fingers crossed that 78b won't freeze over):

- 78b => BLT1850 starting from 75 at year 99
- 78b_4co2 => 4xco2 starting from 75 at year 99
- 78b_1%co2 => 1%co2 starting from 75 at year 99

**Comment by cecilehannay** [comment]
This is a follow up on the discussion from the co-chair meeting.

At the co-chair, we discussed the following slide. 
![Screen Shot 2024-02-20 at 2 17 38 PM](https://github.com/NCAR/amwg_dev/assets/9723220/941bcbbf-751d-40c2-966a-2a5aad6817c8)

I had some concerns about the 78b warming by 0.4K over a 20-year period. It was pointed out that previous runs had similar warming.

I added 54 to the plot and indeed, it seems that 54 experienced similar warming over similar time period. Maybe, my concerns are not relevant and everything is fine after all. 
 
![Screen Shot 2024-02-20 at 2 26 08 PM](https://github.com/NCAR/amwg_dev/assets/9723220/3b523039-6c12-411c-8cc9-ca3be0843545)

I am extending:
78b (#497) => BLT1850 starting from 75 at year 99
78b_4co2  (#498) => 4xco2 starting from 75 at year 99
78b_1%co2  (#499) => 1%co2 starting from 75 at year 99

We will reassess the situation when we have more years.

**Comment by cecilehannay** [comment]
Update on the climate sensitivity runs. 

I added cesm2 to the timeseries and Gregory plots as a reference:
![Screen Shot 2024-02-26 at 10 05 06 AM](https://github.com/NCAR/amwg_dev/assets/9723220/774d126e-fbed-41b0-a46b-4f1926b327b0)

Note that the ECS for 78b in the Gregory plot will change as the runs are getting longer:
<img width="929" alt="Screen Shot 2024-02-25 at 11 29 07 AM" src="https://github.com/NCAR/amwg_dev/assets/9723220/157fff25-7aa5-45eb-bee6-ed68beaa0cd2">

**Comment by cecilehannay** [comment]
Here is the TCR for 78b and cesm2:
```
TCR(78b)   = 1.8 K
TCR(cesm2) = 2.0 K
```

![Screen Shot 2024-02-27 at 10 07 25 AM](https://github.com/NCAR/amwg_dev/assets/9723220/6a2ca49f-a745-4d11-b26a-27e764e536fd)

I am computing the anomalies from a 150-year control for cesm2 and 79-year control for 78b. So the TCR might slightly different if we increase the length of the control.  

**QUESTION: Do we want to extent the 1xco2 further?** 
(@PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @brianpm, @swrneale, @islasimpson @dlawrenncar)

**Comment by PeterHjortLauritzen** [comment]
I would say no as I don't think the numbers will change massively (we will soon be updating to a new CESM tag and turn on RRTMG-P so these numbers may change in the near future ...)

**Comment by cecilehannay** [comment]
Here is the updated timeseries (also adding cesm2 with @jiang-zhu's tuning). 

![Screen Shot 2024-02-27 at 2 10 41 PM](https://github.com/NCAR/amwg_dev/assets/9723220/5a46b363-9872-487e-8d43-8ea1ca83bf3d)

@klindsay28
